operator/config/overlays/cnsa/kustomization.yaml

```diff
@@ -36,7 +36,7 @@ patches:
                   - name: CSI_PROVISIONER_IMAGE
                     value: cp.icr.io/cp/spectrum/scale/csi/csi-provisioner@sha256:893e37a388e7a7463d6c3523311b28cfbc5ae536dbef35430eed272cdc6850dc
                   - name: CSI_LIVENESSPROBE_IMAGE
-                    value: cp.icr.io/cp/spectrum/scale/csi/livenessprobe@sha256:2b10b24dafdc3ba94a03fc94d9df9941ca9d6a9207b927f5dfd21d59fbe05ba0
+                    value: cp.icr.io/cp/spectrum/scale/csi/livenessprobe@sha256:4dc0b87ccd69f9865b89234d8555d3a614ab0a16ed94a3016ffd27f8106132ce
```

Comment:
Before merging the PR, this image has to be pushed to artifactory. Or we can revert the change in cnsa/kustomization.yaml and make this change only on the day of build delivery.

Reply:
True, I am already checking with @hemalathagajendran to push these images to artifactory.